clouddrift/adapters/subsurface.py

```diff
@@ -0,0 +1,170 @@
+"""
+This module defines functions used to adapt the subsurface float trajectories as
+a ragged-array dataset.
+
+The dataset is hosted at https://www.aoml.noaa.gov/phod/float_traj/index.php
+
+Example
+-------
+>>> from clouddrift.adapters import subsurface
+>>> ds = subsurface.to_xarray()
+"""
+
+from datetime import datetime
+import numpy as np
+import os
+import pandas as pd
+import scipy.io
+import tempfile
+import urllib.request
+import xarray as xr
+import warnings
+
+SUBSURFACE_FLOAT_DATA_URL = (
+    "https://www.aoml.noaa.gov/phod/float_traj/files/allFloats_12122017.mat"
```

Comment:
a .mat file is downloaded? How do you read this in python?

Reply:
With scipy. It's there a few lines below.